debezium-embedded/README.md

```diff
@@ -94,7 +94,7 @@ The next few lines define the fields that are specific to the connector, which i
 
 Here, we set the name of the host machine and port number where the MySQL database server is running, and we define the username and password that will be used to connect to the MySQL database. Note that for MySQL the username and password should correspond to a MySQL database user that has been granted the [`REPLICATION SLAVE` privilege](http://dev.mysql.com/doc/refman/5.7/en/replication-howto-repuser.html), allowing the database to read the server's binlog that is normally used for MySQL replication.
 
-The configuration also includes a numeric identifier for the `server.id`. Since MySQL's binlog is part of the MySQL replication mechanism, in order to read the binlog the `MySqlConnector` instance must join the MySQL server group, and that means this server ID must be [unique within all processes that make up the MySQL server group](http://dev.mysql.com/doc/refman/5.7/en/replication-howto-masterbaseconfig.html) and is any integer between 1 and (2^32)−1. In our code we set it to a fairly large but somewhat random value we'll use only for our application.
+The configuration also includes a numeric identifier for the `server.id`. Since MySQL's binlog is part of the MySQL replication mechanism, in order to read the binlog the `MySqlConnector` instance must join the MySQL server group, and that means this server ID must be [unique within all processes that make up the MySQL server group](http://dev.mysql.com/doc/refman/{mysql-version}/en/replication-howto-masterbaseconfig.html) and is any integer between 1 and (2^32)−1. In our code we set it to a fairly large but somewhat random value we'll use only for our application.
```

Comment:
We introduced the `{mysql-version}` variable in this paragraph but the prior explicitly specifies `5.7`.  I'd suggest this change be omitted and instead logged & applied as a separate task where we apply this to all references.

Reply:
I checked the link and fixed it everywhere/all references. yes, this is housekeeping, but imo it will make more work for me now to revert than to leave it like it is now?